inference-engine/tests/ie_test_utils/common_test_utils/common_utils.hpp

```diff
@@ -82,4 +82,16 @@ std::vector<std::pair<std::vector<size_t>, std::vector<elementTypeVector>>>
     return resVec;
 }
 
+template <typename master, typename slave>
```

Comment:
Looks like duplication of `combineShapes`. Please add template argument to the first function and use it for combination creation